cmd/tailscale/cli/up.go

```diff
@@ -451,15 +455,37 @@ func runUp(ctx context.Context, args []string) error {
 		}
 		if url := n.BrowseToURL; url != nil && printAuthURL(*url) {
 			printed = true
-			fmt.Fprintf(Stderr, "\nTo authenticate, visit:\n\n\t%s\n\n", *url)
-			if upArgs.qr {
+			if upArgs.json {
+				type fields struct {
+					AuthURL string
+					QR      string `json:",omitempty"`
```

Comment:
is this always included, or only with the --qr flag?

Reply:
OH! It's a data URL, not the text output. Also worth mentioning.

Reply:
Documented the field.